plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java

```diff
@@ -518,7 +518,7 @@ public void tearDown()
         // Drop storage tables
         MaterializedResult baseResult = computeActual("show tables in tpch");
         for (MaterializedRow row : baseResult.getMaterializedRows()) {
```

Comment:
Please do work with :

```
system.metadata.materialized_views
```
for obtaining the materialized views to drop.

Please do the changes in `TestIcebergMaterializedViews` in a separate commit.

Reply:
@findinpath  this is not needed anymore - it's removed in this review #11626